bin/build-kubernetes-images.sh

```diff
@@ -11,6 +11,7 @@ fi
 TOPDIR=$(cd "$(dirname "$0")/.." || exit; pwd)
 
 source "$TOPDIR/bin/sparkling-env.sh"
+export SPARK_VERSION=$(grep Spark  "$SPARK_HOME/RELEASE" | cut -d' ' -f2)
```

Comment:
We already have SPARK_VERSION exposed in https://github.com/h2oai/sparkling-water/blob/b67d34a2ba5391cf7eb25397fb84c15f2239a809/bin/sparkling-env.sh#L146 but uses value from gradle.properties.

I guess the problem was with minor version?

We have this line to export the installed SPARK_VERSION https://github.com/h2oai/sparkling-water/blob/b67d34a2ba5391cf7eb25397fb84c15f2239a809/bin/sparkling-env.sh#L126

Maybe we can expose it in sparkling-env as `INSTALLED_SPARK_VERSION` ? WDYT?

Reply:
ok, `installed_spark_version` sounds good to me. Will try to use.

yes, the problem i see is the minor version. IMHO, a user should be able create image of any spark version.

Reply:
Yup, absolutely agree

Reply:
The installed version doesn't contain spark fix version part, just `major.minor`. Should we expect that spark-submit is on class_path or take it from `$SPARK_HOME`

Reply:
The sparkling-env script has method `checkSparkHome` which finds SPARK_HOME, we should follow that logic.

We can create new properties such as INSTALLED_SPARK_MAJOR_MINOR and full version as well

Reply:
ok